crates/cacache/src/get.rs

```diff
@@ -53,12 +53,12 @@ impl Reader {
     ///     let mut str = String::new();
     ///     fd.read_to_string(&mut str).await.expect("Failed to read to string");
     ///     // Remember to check that the data you got was correct!
-    ///     fd.check()?;
+    ///     fd.check().await?;
```

Comment:
Since the `IntegrityChecker` is a member of the internal reader, and the internal reader is now wrapped in an `Unblock`, accessing the checker via `self.reader.into_inner()` requires an `.await`. This seemed reasonable for the async API, but it is a breaking change – let me know if you'd prefer I find another way to achieve this!